admin/class-admin.php

```diff
@@ -581,12 +581,16 @@ public function filter_stopwords_from_slug( $slug, $post_title ) {
 	 * @return array
 	 */
 	public function filter_settings_pages( array $pages ) {
+		$pages_to_hide = WPSEO_Advanced_Settings::get_advanced_pages();
```

Comment:
It might be better to put these two assignments below the `wpseo_advanced_settings_enabled ` if-statement.